lib/ssh_scan/database.rb

```diff
@@ -0,0 +1,26 @@
+# API Databases
+require 'ssh_scan/database/mongo'
+require 'ssh_scan/database/sqlite'
+
+module SSHScan
+  class DatabaseConfig
+    def initialize(opts = {})
+      @@db_type = opts['database']['type'] || 'mongodb'
+    end
+
+    def self.from_config_file
+      @@db_path = './lib/ssh_scan/database/database_config.yml'
```

Comment:
Sounds good, let's create an issue or just start doing it in this PR and ask others to follow.